Source/Atj/data/mvp_scenario.json

```diff
@@ -41,6 +41,20 @@
         "hacky_sack_club_flyer_1"
     ],
     "triggers": [
+        {
+            "name": "zach_mood_1_trigger",
+            "all_of": [
+                {
+                    "type": "npc_mood_check",
+                    "npc": "sara",
+                    "check": "LTorEQ",
+                    "value": "2"
+                }
+            ],
+            "actions": [
+                "bind_zak_to_decrement_mood_zak_routine_action"
```

Comment:
Does this action exist?

Reply:
This is a legacy from when I was trying to building all of the mood checks as their own routines, rather than their own action. I thought I had pulled all of that out of the JSON, you found a zombie! Thank you.

Reply:
I do have a green squiggle in this area, the final '}', so please confirm that is all good when you next review
```
            "all_of": [
                {
                    "type": "npc_mood_check",
                    "npc": "sara",
                    "check": "LTorEQ",
                    "value": "2"
                }
        }
```

Reply:
That squiggle is because of the missing closing `']` for `all_of`